packages/backend/src/modules/tvl/ArbitrumTvl.ts

```diff
@@ -40,6 +53,11 @@ export function createArbitrumTvlSubmodule(
 
   // #endregion
 
+  const _arbitrumBalanceProvider = new ArbitrumBalanceProvider(
```

Comment:
there is no need to init it now

Reply:
https://github.com/l2beat/l2beat/pull/1645/commits/64c3141acd75d848e7bf61374483f7c68418dd2e